plugins/scaffolder-backend/src/scaffolder/actions/builtin/catalog/fetchMultiple.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { CatalogApi } from '@backstage/catalog-client';
+import { createTemplateAction } from '@backstage/plugin-scaffolder-node';
+import yaml from 'yaml';
+
+const id = 'catalog:fetchMultiple';
```

Comment:
```suggestion
const id = 'catalog:fetch-multiple';
```

Feeling this fits a bit better with the established naming patterns. Another option is `catalog:fetch:multiple` but that'd be more if it's the specific type of fetch, and in this case I feel it's more of a different way to fetch.